control-plane/csi-driver/src/context.rs

```diff
@@ -492,6 +507,7 @@ pub struct CreateParams {
     encrypted: Option<bool>,
     pvc_name: Option<String>,
     pvc_namespace: Option<String>,
+    pool_allocation_unit_size: Option<u64>,
```

Comment:
Should use cluster_size since we've kept that naming, otherwise with openapi/CLI and SC with different terminology will be confusing.
We'd just need to document what cluster_size means somewhere

```suggestion
    pool_cluster_size: Option<u64>,
```

Reply:
> Should use cluster_size since we've kept that naming, otherwise with openapi/CLI and SC with different terminology will be confusing. We'd just need to document what cluster_size means somewhere

modified